src/shared/cgroup-setup.c

```diff
@@ -674,51 +672,38 @@ int cg_attach_many_everywhere(CGroupMask supported, const char *path, Set* pids,
         return r;
 }
 
-int cg_migrate_everywhere(CGroupMask supported, const char *from, const char *to, cg_migrate_callback_t to_callback, void *userdata) {
+int cg_migrate_controllers(CGroupMask supported, CGroupMask mask, const char *from, cg_migrate_callback_t to_callback, void *userdata) {
         CGroupController c;
         CGroupMask done;
-        int r = 0, q;
-
-        if (!path_equal(from, to))  {
-                r = cg_migrate_recursive(SYSTEMD_CGROUP_CONTROLLER, from, SYSTEMD_CGROUP_CONTROLLER, to, CGROUP_REMOVE);
-                if (r < 0)
-                        return r;
-        }
-
-        q = cg_all_unified();
-        if (q < 0)
-                return q;
-        if (q > 0)
-                return r;
+        assert(to_callback);
```

Comment:
i think this function should have a comment explaining what it does, i.e. that it only and exclusively migrates the processes in the cgroupsv1 controller hierarchies, i.e. not in the name=systemd hierarchy.

Reply:
Should be clear from the name now.